<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="onebill-VPCSE17GG" failures="1" tests="1" name="Test1" time="42.156" errors="0" timestamp="2021-07-23T17:28:17 IST">
  <testcase classname="Generic.LOGOUT" name="Logout" time="0.946">
    <failure type="java.lang.AssertionError" message="Logout is Not Done Successfully  expected [Login | Salesforce] but found [Home | Salesforce]">
      <![CDATA[java.lang.AssertionError: Logout is Not Done Successfully  expected [Login | Salesforce] but found [Home | Salesforce]
at Generic.LOGOUT.Logout(LOGOUT.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Logout -->
</testsuite> <!-- Test1 -->
